astroplan/scheduling.py

```diff
@@ -293,11 +448,15 @@ def __call__(self, oldblock, newblock, start_time, observer):
             from .constraints import _get_altaz
             from astropy.time import Time
 
-            aaz = _get_altaz(Time([start_time]), observer, [oldblock.target, newblock.target])['altaz']
-            # TODO: make this [0] unnecessary by fixing _get_altaz to behave well in scalar-time case
-            sep = aaz[0].separation(aaz[1])[0]
-            
-            components['slew_time'] = sep / self.slew_rate
+            if type(oldblock) == ObservingBlock:
+                aaz = _get_altaz(Time([start_time]), observer, [oldblock.target, newblock.target])['altaz']
+                # TODO: make this [0] unnecessary by fixing _get_altaz to behave well in scalar-time case
+                sep = aaz[0].separation(aaz[1])[0]
+                components['slew_time'] = sep / self.slew_rate
+            elif type(oldblock) == TransitionBlock:
+                # if the previous block was a TransitionBlock, then assume slew
+                # hapens during that block
+                components['slew_time'] = 0.
```

Comment:
Should probably figure out the slew time and then use that to decide if the transition block was long enough, right?